tests/queries/models.py

```diff
@@ -737,3 +737,57 @@ class RelatedIndividual(models.Model):
 
     class Meta:
         db_table = 'RelatedIndividual'
+
+
+class SchemaQualified(models.ModelTable):
+    def __init__(self, schema, table):
+        self.schema = schema
+        self.table = table
+
+    def as_sql(self, compiler, connection):
+        # Note that it would be easy to change the schema using thread locals
+        # if wanted.
```

Comment:
Migrations are only a part of it (and I've been thinking of ways around the "set search_path, migrate, repeat" cycle, which really does take a long time with multiple schemata.

With this approach, we could at least rewrite the DDL query (or the DML query if it was a data migration, but that's even harder to deal with), and possibly make it faster.

I'm not currently writing production code against my solution anyway, so it's all about the future...